B2b branding phase 1 (#7802)
auth0/docs: articles/architecture-scenarios/_includes/_branding/_custom-domain-naming.md

In addition, Custom Domain functionality offers you complete control over the [certificate management](/custom-domains#certificate-management) process. By default, Auth0 provides standard SSL certificates, but if you configure a custom domain, you can use Extra Validation (EV) SSL certificates or similar to provide the visual, browser-based cues that offer your visitors additional peace of mind.

In general, we see customers having the most success when they use a centralized domain for authentication - this is especially the case if the company offers multiple products or service brands. By using a centralized domain, you can provide end users with a consistent user experience while also minimizing the need to maintain multiple production tenants in Auth0.

<% if (platform === "b2b") { %>
::: warning

> If your organization users will all be isolated from each other and you **require** that those users are presented a login page at a custom domain, then your only option is to create a [separate tenant for each organization](/architecture-scenarios/b2b/b2b-architecture#tenant-provision-for-complex-organizations).

Reviewer: ```suggestion
If your organization users will all be isolated from each other and you **require** that those users are presented a login page at a custom domain, then your only option is to create a [separate tenant for each organization](/architecture-scenarios/${platform}/${platform}-architecture#tenant-provision-for-complex-organizations).
```

> If your organization users will all be isolated from each other and you **require** that those users are presented a login page at a custom domain, then your only option is to create a [separate tenant for each organization](/architecture-scenarios/${platform}/${platform}-architecture#tenant-provision-for-complex-organizations).

Reviewer 2: ```suggestion
If your customer organizations will be isolated from each other, and you **require** that users are presented a login page via a custom domain URL (for each organization), then your only option is to create a [separate tenant for each organization](/architecture-scenarios/${platform}/${platform}-architecture#tenant-provision-for-complex-organizations).
```

Author: Added to separate PR 7818
